.github/project.yml

```diff
@@ -1,5 +1,5 @@
 name: Quarkiverse MyBatis Extension
 release:
   current-version: 0.0.2
-  next-version: 0.0.3-SNAPSHOT
+  next-version: 0.0.3
```

Comment:
Please don't change the version here. And the quarkiverse bot will help us for releasing the new version.